### PR TITLE
orchagent: Changing select time out to 1s

### PR DESCRIPTION
**Pull request by stcheng**

### CODE REVIEW

File: orchagent/orchdaemon.cpp [hunk 1]
```diff
@@ -83,7 +83,7 @@ void OrchDaemon::start()
         Selectable *s;
         int fd, ret;
 
-        ret = m_select->select(&s, &fd, 1);
```
Comment (lguohan) [inline orchagent/orchdaemon.cpp]: can you explain the reason to reduce the select timeout?

Comment (qiluo-msft) [inline orchagent/orchdaemon.cpp]: If it is a valid usage, we should fix inside the select.

Comment (lguohan) [inline orchagent/orchdaemon.cpp]: what is the reason to have timeout? why do we have timeout?

Comment (stcheng) [inline orchagent/orchdaemon.cpp]: this timeout allows the merging time for the m_toSync map to merge and allows the time for items with unresolved dependencies to be resolved without retrying too many times

Comment (lguohan) [inline orchagent/orchdaemon.cpp]: suggest to increase the timeout to 1 second.

**Comment by stcheng** [comment]
updated. the reason of having this timeout is to 
1. reduce the speed of retrying logic when one orchagent task is not able to be done and needs a second try.
2. have a time slot for the tasks in the m_toSync queue to merge and reduce the tasks to be done

**Pull request merged**